docs/guides/cases/connect-http-device-via-gateway.md

```diff
@@ -0,0 +1,141 @@
+---
+title: Connect a HTTP Device via Gateway
+sidebar_position: 7
+---
+
+# Connect a HTTP Device via Gateway
+
+## Introduction to LwM2M Gateway
+
+The LwM2M (Lightweight Machine to Machine) Gateway consists of two main components: an LwM2M client that connects to the LwM2M server and an HTTP client that connects to deviceShifu.  The LwM2M Gateway enables deviceShifu to adapt to the LwM2M protocol, handle requests from the LwM2M server, and push data to the cloud, fulfilling the two major functions required by the LwM2M protocol: the capability to pull data from the device and post data from the cloud to the device.
+
+## Get the Example and Deploy LwM2M deviceShifu
+
+We'll demonstrate the integration using an HTTP-based thermometer as an example. You can find the corresponding templates in [lwm2m_gw_http](https://github.com/Edgenesis/shifu/tree/main/examples/lwm2m_gw_http). These templates can be modified to integrate other HTTP protocol devices. To ensure our Gateway operates correctly, we need to modify several configuration files.
```

Comment:
please organize words simple and clearly containing two key messages:
1. this is an HTTP deviceshifu
2. this deviceshifu will have a LwM2M gateway
to meet these two criterias, need to following steps of yaml configuration